src/languages/es.js

```diff
@@ -430,6 +430,7 @@ export default {
         twoFactorCode: 'Autenticación de 2 factores',
         requiredWhen2FAEnabled: 'Obligatorio cuando A2F está habilitado',
         error: {
+            incorrectPassword: 'Contraseña incorrecta. Por favor inténtalo de nuevo',
```

Comment:
Is there supposed to be a full stop at the end? I think @pecanoro can confirm this.

Reply:
Yes, I would say so since the English version has the full stop at the end.